### PR TITLE
Added Ceres' Lance gear to the antagonist uplink

### PR DESCRIPTION
**Pull request by Generalcamo**
Ceres' Lance is a PMC faction dedicated to hunting down and capturing (or destroying) rogue synthetics. They are affiliated with NanoTrasen and Biesel, but have also been known to work with the Sol Alliance to hunt down these dangerous units.

Now you can play them as a mercenary. Your weapons are designed primarily to neutralize or destroy rogue synthetic units. If your standard weaponry fails, you also have an anti-material rifle, however unlike the Hegemony crate this is the full-size rifle. You only have four bullets available as a result instead of twelve, so make them count. All that fails, and you have force gloves to wrestle the synthetic down with. 

This faction has unathi gear available in the voidsuits.

### CODE REVIEW

**Comment by Generalcamo** [comment]
!review

**Comment by scheveningensun** [comment]
Noting out that there are some balance issues with this PR, namely that the bunker suit was event only and was overtuned specifically to avoid players trying to spotlight the event by attempting to kill them.

image
Namely, the bunker suit has equivalent armor values in melee and laser for the heavy corp armor, and to the specialized suits respectively for ballistic and laser damage. You can completely offset the movement speed penalty with hyperzine and then run a personal shield along with this.

This probably also needs lore team approval too, as Ceres Lance has not been active for years in lore terms.

**Comment by Generalcamo** [comment]
> You can completely offset the movement speed penalty with hyperzine and then run a personal shield along with this.
Going to go on a tangent to say that this is a problem with hyperzine, not the armor itself.

> Namely, the bunker suit has equivalent armor values
To the Elyran battlesuit, which is currently available as a ninja outfit.

I will note that Kyres requested I remove the bunker suit, as he would prefer it return to extremely high values for future lore events. I was indeed planning to do this, and allow the voidsuits only for the mercs.

File: code/game/objects/structures/crates_lockers/crates/gear_loadout.dm [hunk 1]
```diff
@@ -755,6 +755,50 @@
...
+	new /obj/item/voidsuit_modkit/ceres_lance_unathi(src)
+	new /obj/item/voidsuit_modkit/ceres_lance_unathi(src)
+	new /obj/item/voidsuit_modkit/ceres_lance_unathi(src)
+	new /obj/item/gun/energy/rifle/ionrifle(src)
+	new /obj/item/gun/energy/rifle/pulse(src)
+	new /obj/item/gun/energy/rifle/pulse(src)
+	new /obj/item/gun/energy/pulse/pistol(src)
+	new /obj/item/gun/energy/pulse/pistol(src)
+	new /obj/item/gun/energy/pulse/pistol(src)
+	new /obj/item/gun/energy/pulse/pistol(src)
```
Comment (NonQueueingMatt) [inline code/game/objects/structures/crates_lockers/crates/gear_loadout.dm:787]:
```/obj/item/projectile/beam/pulse
	name = "pulse"
	icon_state = "u_laser"
	damage = 50
	armor_penetration = 50
```

Yeah let's not add a guaranteed 50 damage beam to normal gameplay please. Remove the pulse rifles and pistols.

File: code/game/objects/structures/crates_lockers/crates/gear_loadout.dm [hunk 2]
```diff
@@ -755,6 +755,50 @@
...
+	new /obj/item/clothing/gloves/force/basic(src)
+	new /obj/item/clothing/gloves/force/basic(src)
+
+
+/obj/structure/closet/crate/secure/gear_loadout/ceres_lance/single/fill()
+	new /obj/item/clothing/under/lance(src)
+	new /obj/item/clothing/suit/space/void/lancer(src)
+	new /obj/item/clothing/head/helmet/space/void/lancer(src)
+	new /obj/item/voidsuit_modkit/ceres_lance_unathi(src)
+	new /obj/item/gun/energy/rifle/pulse(src)
```
Comment (NonQueueingMatt) [inline code/game/objects/structures/crates_lockers/crates/gear_loadout.dm:799]: See above

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

**Comment by Generalcamo** [comment]
Removed the pulse stuff and bunker gear.

Replaced them with more standard weaponry, along with the anti-material rifle (as lore states they use this) unique to this crate. Unlike the hegemony crate, this is a full-size model, but this is compensated by reducing the bullet count from 12 to 4. Make those shots count. 

If all of this fails, you have standard force gloves instead of basic force gloves. This is to compensate for the removal of the hardsuit from the crate however.

File: code/game/objects/structures/crates_lockers/crates/gear_loadout.dm [hunk 3]
```diff
@@ -755,6 +755,56 @@
...
+	new /obj/item/voidsuit_modkit/ceres_lance_unathi(src)
+	new /obj/item/voidsuit_modkit/ceres_lance_unathi(src)
+	new /obj/item/voidsuit_modkit/ceres_lance_unathi(src)
+	new /obj/item/voidsuit_modkit/ceres_lance_unathi(src)
+	new /obj/item/gun/energy/rifle/ionrifle(src)
+	new /obj/item/gun/projectile/heavysniper(src)
+	new /obj/item/ammo_casing/a145(src)
+	new /obj/item/ammo_casing/a145(src)
+	new /obj/item/ammo_casing/a145(src)
+	new /obj/item/ammo_casing/a145(src)
```
Comment (NonQueueingMatt) [inline code/game/objects/structures/crates_lockers/crates/gear_loadout.dm:785]: No to this too. This weapon is out of gameplay for good reason too; it's an instant-kill through walls. Even worse with thermals.

Make it a scout rifle or a PEAC.

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

**Pull request closed without merging**